hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java

```diff
@@ -984,138 +1154,46 @@ public void handle(
           dryRun = true;
 
         } else if (option.equals("-I")) {
-          if (args.length < 1) {
+          if (args.size() < 1) {
```

Comment:
Nit: Can be addressed here or in separate PR.

This looks to be an existing issue, but I think this check is not right. `-I` does not itself require any additional arguments. I think we should remove this if check here.

Reply:
good catch, thanks!